src/store/wallet.ts

```diff
@@ -1,18 +1,18 @@
-import { ZK_API_BASE, ZK_NETWORK } from "@/plugins/build";
+import { ZK_API_BASE } from "@/plugins/build";
 import onboardConfig from "@/plugins/onboardConfig";
 import utils from "@/plugins/utils";
 import { walletData } from "@/plugins/walletData";
 import watcher from "@/plugins/watcher";
 import web3Wallet from "@/plugins/web3";
-import { BalancesList, iWalletData, ZkInBalance, ZkInFeesObj, ZkInTx, ZkInWithdrawalTime, ZKTypeDisplayBalances, ZKTypeDisplayToken } from "@/types/lib";
+import { BalancesList, ZkInBalance, ZkInFeesObj, ZkInTx, ZkInWithdrawalTime, ZKTypeDisplayBalances, ZKTypeDisplayToken } from "@/types/lib";
 import { ExternalProvider } from "@ethersproject/providers";
 import Onboard from "@matterlabs/zk-wallet-onboarding";
 import { API } from "@matterlabs/zk-wallet-onboarding/dist/src/interfaces";
 import { BigNumber, BigNumberish, ethers } from "ethers";
 import { actionTree, getterTree, mutationTree } from "typed-vuex";
 import { provider } from "web3-core";
-import { closestPackableTransactionFee, getDefaultProvider, Provider, Wallet } from "zksync";
-import { Address, Fee, Network, TokenSymbol } from "zksync/build/types";
+import { closestPackableTransactionFee, Wallet } from "zksync";
+import { Address, Fee, TokenSymbol } from "zksync/build/types";
 
 interface feesInterface {
```

Comment:
@JackHamer09 all types should follow the prefix semantics e.g. ```ZKI``` for the interface, ```ZKType``` for the types. Each type and interface should be moved to the ```@/types/lib.d.ts```